documentation.md

```diff
@@ -0,0 +1,117 @@
+## Appendix C - Documentation
+
+In this section of the Appendix, we attempt to highlight at a high level, all the decisions that were taken during the entirety of this capstone project.
+
+### Data Augemntations 
+
+The albumentations library is powered by PyTorch for image transformations. 
+
+To use the albumentations library:
+
+- This [guide](https://albumentations.ai/docs/examples/pytorch_classification/) is extremely helpful for our classification task. You develop your own class that inherits from the torch dataset. To see an example of implementation for our dataset, please check this [notebook](https://github.com/UBC-MDS/capstone-gdrl-lipo/blob/auto_exp1/notebooks/autoalbument.ipynb).
+- Some transformations that believed to  be helpful - [here] (https://github.com/albumentations-team/albumentations#pixel-level-transforms) is the entire list:
+- CLAHE
+- Downscale
+- Equalize 
+- Hue Saturation (may not apply if no color channels)
+- ISO noise 
+- Invert Image
+- Random Birghtness
+- Random Contrat 
+- Random Snow
+- Shift Scale rotate 
```

Comment:
I think most of these details can be taken out in terms of listing them here. We can that we initially used a [web interface](https://albumentations-demo.herokuapp.com/) provided by the albumentations team that allowed us to see what the different transformations looked like and based on that, we chose some transformations to test in our pipeline. The final model was chosen based on accuracy score.